homeassistant/components/youless/sensor.py

```diff
@@ -0,0 +1,197 @@
+"""The sensor entity for the Youless integration."""
+from __future__ import annotations
+
+from youless_api.youless_sensor import YoulessSensor
+
+from homeassistant.components.youless import DOMAIN
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_DEVICE, DEVICE_CLASS_POWER
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Initialize the integration."""
+    coordinator = hass.data[DOMAIN][entry.entry_id]
+    device = entry.data[CONF_DEVICE]
+    if device is None:
+        device = entry.entry_id
+
+    async_add_entities(
+        [
+            GasSensor(coordinator, device),
+            PowerMeterSensor(coordinator, device, "low"),
+            PowerMeterSensor(coordinator, device, "high"),
+            PowerMeterSensor(coordinator, device, "total"),
+            CurrentPowerSensor(coordinator, device),
+            DeliveryMeterSensor(coordinator, device, "low"),
+            DeliveryMeterSensor(coordinator, device, "high"),
+            ExtraMeterSensor(coordinator, device, "total"),
+            ExtraMeterSensor(coordinator, device, "usage"),
+        ]
+    )
+
+
+class YoulessBaseSensor(CoordinatorEntity, Entity):
```

Comment:
We don't need to inherit `Entity` when inheriting `CoordinatorEntity`.

Please also inherit `SensorEntity` imported from the sensor integration.